test/new-e2e/tests/otel/otel-agent/config/loadbalancing.yml

```diff
@@ -0,0 +1,131 @@
+receivers:
```

Comment:
Nice! 🍰 

Smart way to use a single collector retro-feeding itself the load-balanced output and then directing to the fake intake.